src/ApiProblem/ApiProblem.php

```diff
@@ -135,7 +135,9 @@ public static function fromException(Exception $exception, $title = '', $type =
             $type = self::RFC2616;
         }
 
-        return new self($code, $exception->getMessage(), $title, $type, $additionalDetails);
+        $detail = empty($exception->getMessage()) ? get_class($exception) : $exception->getMessage();
```

Comment:
I've got to agree, the getMessage() approach may lead to the same situation.

@mbabjak let me think about it